client/app/views/index.js

```diff
@@ -1,57 +1,60 @@
 'use strict';
 
-var _ = require('lodash');
-var h = require('../helpers/helpers');
+var _ = require('../helpers/utils');
```

Comment:
what is the idea behind this? not using lodash and using the utils instead :confused: